release-please-config.json

```
@@ -0,0 +1,13 @@
+{
+    "release-type": "simple",
+    "bump-minor-pre-major": true,
+    "bump-patch-for-minor-pre-major": true,
+    "include-component-in-tag": true,
+    "separate-pull-requests": true,
+    "sequential-calls": true,
+    "packages": {
+        "releases-maxa-ai": {
+            "component": "releases-maxa-ai"
+        }
```

Do we want to increase the `version` in the `package.json` ?

```suggestion
        "releases-maxa-ai": {
            "component": "releases-maxa-ai",
            "release-type": "node"
        }
```

---

It's a good idea. I should also update the name in that file as well, then!